types/pdf-thumbnail/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
```

Comment:
Sorry I didn’t notice this in my earlier review—I think this can be removed now that you’ve fixed the source, right?

Reply:
Oops, missed it too... Fixed!